lib/david/registry.rb

```diff
@@ -21,8 +21,17 @@ def observe
       Celluloid::Actor[:observe]
     end
 
-    def server
-      Celluloid::Actor[:server]
+    def server_udp
```

Comment:
The `server_udp` and `server_dtls` functions are only used by the `Registry#servers` function. They could those be marked as private or directly integrated into the `#servers` function.